### PR TITLE
add alternative notes for open dialogue page

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

File: ckanext/recombinant/logic.py [hunk 1]
```diff
@@ -259,7 +259,8 @@ def _dataset_fields(geno):
     """
     return the dataset metadata fields created for dataset definition geno
     """
-    return {'title': geno['title'], 'notes': geno.get('notes', '')}
+    notes = geno.get('alt_notes', geno.get('notes', ''))
+    return {'title': geno['title'], 'notes': notes}
```
Comment (wardi) [inline ckanext/recombinant/logic.py]: this seems strange. Why replace the notes with alt_notes?

This field gets stored on the dataset itself and will appear in the dataset search page on the registry. The text you've used in ckanext-canada doesn't seem appropriate.

File: ckanext/recombinant/templates/recombinant/resource_edit.html [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
-{% block page_title %}<h1 id="wb-cont" property="name">{{ _(resource.description) }}</h1>{% endblock %}
+{% set geno = h.recombinant_get_geno(dataset.dataset_type) %}
+{% block page_title %}<h1 id="wb-cont" property="name">{{ _(resource.description) }}</h1>
+  {% if 'alt_notes' in geno %}
+    <div class="alert alert-info" role="alert">
+       {{ _(geno.alt_notes) }}
+    </div>
+  {% endif %}
+{% endblock %}
+
```
Comment (wardi) [inline ckanext/recombinant/templates/recombinant/resource_edit.html]: Why not move this to ckanext-canada instead? There's nothing reusable here.

File: ckanext/recombinant/logic.py (hunk 1 above)
Comment (fanjinfei) [inline ckanext/recombinant/logic.py]: In gitlib issue 7, they want `replace the record description in the search pages`.

**Comment by fanjinfei** [comment]
use temporary solutions

**Pull request closed without merging**